tests/testthat/data/helper_generate_stochastic_sir_rt.R

```diff
@@ -0,0 +1,8 @@
+pkgload::load_all()
```

Comment:
what is this here for? I think I would prefer to expect the dev to write this?

Reply:
I see why you are doing this but generally I have found having data snapshots used in tests to be quite brittle. It might be nice to have an issue to explore if you can get the same testing using a on the fly generated dataset (i.e in setup.R)

Reply:
#98